frontend/src/app/home/Faqs.tsx

```
@@ -0,0 +1,75 @@
+        <div className="p-4 rounded-lg shadow-xl pb-8 mt-8">
+          <div className="text-center">
+            <h1 className="text-5xl font-bold tracking-tight text-white sm:text-5xl">
+              FAQs
+            </h1>
+            <p className="mt-4 text-lg leading-8 text-gray-300">
+              Here are some of the frequently asked questions
+            </p>
+          </div>
+          <div className="space-y-12 px-2 xl:px-16 mt-16">
+            {faqItems.map((faqItem) => (
+              <FaqItem faqItem={faqItem} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)